packages/react-static/package.json

```diff
@@ -27,6 +27,7 @@
     "publishLink": "echo '{{event}} to {{changed}}' && yalc publish"
   },
   "peerDependencies": {
+    "postcss": "^8.2.4",
```

Comment:
this may be more involved - looks like several other deps depend on ^7.0.0 such as the css-loader :disappointed:

Reply:
There "just" peer-dependencies tho. It's okay to upload it accross the board, or do it incrementally. You can change it to `>= 7.0.0, < 9.0.0` for example.

Reply:
that is a good point, with the upgrade of postcss-loader incrementally getting the other deps are easier and this is only upgrading the loader :slightly_smiling_face: